lib/pbench/server/auth/__init__.py

```diff
@@ -366,15 +388,18 @@ def _get(self, path: str, **kwargs) -> requests.Response:
                 verify=self.verify,
             )
         except Exception as exc:
-            self.logger.exception("{}", str(exc))
-            raise OidcError(
+            self.logger.exception(
+                "Exception submitting a GET request for OIDC client {}", self.__repr__()
+            )
+            raise OpenIDClientError(
                 HTTPStatus.INTERNAL_SERVER_ERROR, f"Can't connect to server {exc}"
             )
```

Comment:
Well, I just used very generic language at the time I wrote this without thinking too much about wording. I am catching a generic `Exception` so that should basically catch everything and the exception message should also be present to the caller since we are re-raising the same exception as an `OpenIDClientError`. Maybe I should change it to something like "Failure to complete the GET request from the OpenID client"